src/index.html

```diff
@@ -15,8 +15,105 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
```

Comment:
The same issue with the closing bracket as in the previous comments. The closing bracket on line 21 should be on the same level as the opening bracket on line 18.